app/models/service_template_ansible_playbook.rb

```diff
@@ -157,6 +155,7 @@ def update_catalog_item(options, auth_user = nil)
   end
 
   def destroy
```

Comment:
We should not override `destroy` method. Instead use the `after_destroy` hook.  You can rename `destroy` to `delete_job_templates` and declare
```
  after_destroy :delete_job_templates
```